sdk/loadtestservice/Azure.Developer.Playwright/src/Utility/ClientUtilities.cs

```diff
@@ -55,6 +55,16 @@ internal string GetDefaultRunId()
             return runId;
         }
 
+        internal string GetDefaultRunName()
+        {
+            var runNameFromEnvironmentVariable = _environment.GetEnvironmentVariable(Constants.s_playwright_service_run_name_environment_variable);
+            if (!string.IsNullOrEmpty(runNameFromEnvironmentVariable))
+                return runNameFromEnvironmentVariable!;
+            var runId = GetDefaultRunId(); // RunId as the default RunName
```

Comment:
why do we need to calculate it again, i believe your caller function should already have runId, which can sent here as param ??